### PR TITLE
Add isEmpty and isNotEmpty assertions for File

### LINKED ISSUE #1638: Add isEmpty and isNotEmpty assertions for File
#### Summary
I would like to contribute a couple of assertions to check whether the content of a given file is empty or not. These assertion could simply check file.length(), which returns the length of a file in bytes. At present I'm having to do something like `assertThat(file.length()).isGreaterThan(0)`

#### Example
```java
assertThat(file)
.isEmpty()
.isNotEmpty()
```

### PR DESCRIPTION
**Pull request by steveorourke**
#### Check List:
* Fixes #1638
* Unit tests : YES
* Javadoc with a code example (on API only) : YES

### CODE REVIEW

File: src/main/java/org/assertj/core/internal/Files.java [hunk 1]
```diff
@@ -287,6 +288,30 @@ public void assertCanWrite(AssertionInfo info, File actual) {
...
+  /**
+   * Asserts that the given {@code File} is not empty (i.e. size is greater than zero bytes).
+   * @throws AssertionError if the given {@code File} is {@code null}.
+   * @throws AssertionError if the given {@code File} does not exist.
+   * @throws AssertionError if the given {@code File} is empty.
+   */
+  public void assertIsNotEmptyFile(AssertionInfo info, File actual) {
+    assertIsFile(info, actual);
+    if (actual.length() > 0) return;
+    throw failures.failure(info, shouldNotBeEmpty());
```
Comment (joel-costigliola) [inline src/main/java/org/assertj/core/internal/Files.java]: introduce `shouldNotBeEmpty*(File)` with message `"%nExpecting file <%s> not to be empty"` (add the corresponding test in `ShouldNotBeEmpty_create_Test`.

File: src/test/java/org/assertj/core/error/ShouldBeEmpty_create_Test.java [hunk 2]
```diff
@@ -28,18 +28,28 @@
  * @author Alex Ruiz
  * @author Yvonne Wang
  */
-public class ShouldBeEmpty_create_Test {
+@DisplayName("ShouldBeEmpty create")
+class ShouldBeEmpty_create_Test {
```
Comment (joel-costigliola) [inline src/test/java/org/assertj/core/error/ShouldBeEmpty_create_Test.java]: thanks for the refactoring!

**Comment by steveorourke** [comment]
Thanks for the feedback @joel-costigliola. I've updated the PR with the suggested changes.

**Comment by joel-costigliola** [comment]
Integrated thanks @steveorourke!

**Pull request closed without merging**